### DISCUSSION TITLE
How to Retrieve Sequential User ID?

### DISCUSSION THREAD

**Discussion by yenharvey** [logto-io/logto#6773] [Q&A] [answered]
Currently, when retrieving user information via Logto, the `sub` returned is a random string. However, I need a sequential, auto-incremented ID to track the order in which users register. I couldn't find this option in the documentation. Does Logto support such a field, or is there a simpler implementation to achieve this?

**Comment by charIeszhao** [comment, marked as answer]
Maybe you can use the "custom data".
Docs: https://docs.logto.io/docs/references/users/custom-data/

**Reply by yenharvey** [reply]
> Maybe you can use the "custom data". Docs: https://docs.logto.io/docs/references/users/custom-data/

Thanks for your reply! Regarding the use of custom data, I only found this API. If I want to get the maximum ID of the current user, can I only get it through paging? [API link](https://openapi.logto.io/operation/operation-listusers) provides paging, but I am worried that the "infinite" paging method is not efficient enough. In addition, is the ID in the response body a continuous numeric ID or a random string like UserID?

**Reply by yenharvey** [reply]
> Maybe you can use the "custom data". Docs: https://docs.logto.io/docs/references/users/custom-data/

My bad, I didn't notice the hook before.
